tools/cfg-get-cert.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 
-URL="${BASE_URL}/v1/tokenpki/${DEP_NAME}"
+CN="${1:-depserver}"
+VALIDITY_DAYS="${2:-1}"
+URL="${BASE_URL}/v1/tokenpki/${DEP_NAME}?cn=$CN&validity_days=$VALIDITY_DAYS"
```

Comment:
Any reason not to keep it as you had it? If both args are optional they'll just be submitted as empty which should use defaults on the server side. :)

```suggestion
URL="${BASE_URL}/v1/tokenpki/${DEP_NAME}?cn=$1&validity_days=$2"
```